packages/runtime/runtime-definitions/src/garbageCollectionDefinitions.ts

```diff
@@ -44,7 +50,7 @@ export interface IGarbageCollectionData {
 	/**
 	 * The GC nodes of a Fluid object in the Container. Each node has an id and a set of routes to other GC nodes.
 	 */
-	gcNodes: { [id: string]: string[] };
+	gcNodes: { [id: string]: string[] | typeof GarbageCollectionHandle };
```

Comment:
So in the handle case there's no additional info in the "handle" itself, it's just a signal to go look up the GC data by the id?

Reply:
yeah, the ID is built as the GC data moves up the stack. By the time it reached GC, the path is basically the handle.